src/user/introduction.rst

```diff
@@ -9,14 +9,11 @@ A brief introduction
 What is conda-forge?
 --------------------
 
-Conda-forge is a community effort that provides conda packages for a wide range of software.
+``conda-forge`` is a community effort that provides conda packages for a wide range of software.
 
-**Missing a package that you would love to install with conda?** - Chances are we have already packaged it for you!
-
-You can `search <https://anaconda.org/>`__ for packages online. Look out for packages provided by our ``conda-forge`` organization.
-
-**Cannot find a package or only outdated versions of a package?** - Everybody is welcome to contribute to our package stack! Please refer to :ref:`becoming_involved`, for an overview on how to start contributing.
+**Missing a package that you would love to install with conda?** - Chances are we have already packaged it for you! You can `search <https://anaconda.org/>`__ for packages online as well as keep an eye out for packages from our ``conda-forge`` organization.
 
+We value all kinds of contributions- not just code. A few recommended ways to start contributing to conda-forge are– `Contribute new packages <https://conda-forge.org/docs/maintainer/adding_pkgs.html>`_, Help update and `maintain packages <https://conda-forge.org/docs/maintainer/updating_pkgs.html#maintaining-pkgs>`_, Suggest or implement improvements for our `infrastructure <https://conda-forge.org/docs/maintainer/infrastructure.html#infrastructure>`_, and/or Help `improve the documentation <https://conda-forge.org/docs/user/contributing.html#improve-docs>`_.. For a detailed overview please refer to  :ref:`becoming_involved`
```

Comment:
Maybe this series of links reads better as a bullet point list? As in:


```suggestion
We value all kinds of contributions - not just code. A few recommended ways to start contributing to conda-forge are:

 - `Contribute new packages <https://conda-forge.org/docs/maintainer/adding_pkgs.html>`_
 - Help update and `maintain packages <https://conda-forge.org/docs/maintainer/updating_pkgs.html#maintaining-pkgs>`_
 - Suggest or implement improvements for our `infrastructure <https://conda-forge.org/docs/maintainer/infrastructure.html#infrastructure>`_
 - Help `improve the documentation <https://conda-forge.org/docs/user/contributing.html#improve-docs>`_.
 
 For a detailed overview please refer to  :ref:`becoming_involved`.
```

Reply:
Yes. I guess.

Also, another question- may sound silly... But I'll play my newbie card and take my chances. 😅

I believe creating more/maintaining recipes is a contribution as well?

Although it's not in the docs yet... Maybe the person can be redirected to staged recipes?

Reply:
Yes. I guess.

Also, another question- may sound silly... But I'll play my newbie card and take my chances. 😅

I believe creating more/maintaining recipes is a contribution as well?

Although it's not in the docs yet... Maybe the person can be redirected to staged recipes?